### DISCUSSION TITLE
Why is there a pokeball at top left of my reading experience?

### DISCUSSION THREAD

**Discussion by arooni** [koreader/koreader#11839] [Q&A] [answered]
I like Pokémon but why?

**Comment by arooni** [comment]
![20240517_095831](https://github.com/koreader/koreader/assets/556667/c1d6711e-62ab-49b8-b2e3-0c3d63eab948)

**Reply by mergen3107** [reply]
This is i deed called Pokeball :D 
Usually should appear when you long-tap text. 

Try searching by Pokeball in issues

**Comment by jonnyl2** [comment, marked as answer]
The symbol indicates that the time for the very-long-press interval (default: 3 seconds) has elapsed and the system will now show the pop-up dialog, as opposed to apply the selected default option for text selection, when you release the finger. When that dialog box appears, the pokeball should disappear; however, there is a glitch that if you (purposely or inadvertently) touch the screen with a second finger while the pokeball is showing, it will not disappear when you lift the fingers. At the moment, to make it go away, you then have to long-press anywhere on the page.
